### DISCUSSION TITLE
Whether to support platforms other than pc?

### DISCUSSION THREAD

**Discussion by ttttttttttttttllllllllllllll** [Tualin14/baPlayer#27] [Development]
PC: √
Android: Troublesome operation and now we can't add the download game resources function, so importing resources can be a hassle.
WebGL: Will open game resources.

So whether to give up other platforms, only support the PC platform?

According to the official requirements are not allowed to leak the game resources, fortunately as does not need these resources to build.
So as is safe now.
But it may not be safe to go for support for other platforms.

**Comment by kiraio-moe** [comment]
> Android: Troublesome operation and now we can't add the download game resources function, so importing resources can be a hassle.

In the past I have tried working on the Android platform and it was really a mess (inconsistent UI, Android Storage permission error, etc.), so for now I will focus on the PC platform.

> WebGL: Will open game resources.

Can't web apps open local files?

> So whether to give up other platforms, only support the PC platform?

I also don't know if many people will use Aris Studio on that platforms (especially Web).
For now, let's focus on the PC platform first, then if someone requests support for another platform, we'll try to work on it.

**Reply by ttttttttttttttllllllllllllll** [reply]
Sorry for not replying for so long, I have to deal with personal matters and I still can't find the time in the short term.

>Can't web apps open local files?

If we use webgl and use local files, why not just use the PC version?

Some individuals will use the Android version because they are students without computers, but thery will not be able to accept the complex operation of the Android version.

ArisStudio can not be open on the Internet like momotalk, so at present I plan to abandon other platforms and only support the PC version. 
And I'm going to drop the load function for remote resources and change the UnityWebRequest to File to get the file.
